Formula/bracken.rb

```diff
@@ -0,0 +1,28 @@
+class Bracken < Formula
+  # cite Lu_2017: "https://doi.org/10.7717/peerj-cs.104"
+  desc "Bayesian estimation species abundances from Kraken output"
+  homepage "https://ccb.jhu.edu/software/bracken/index.shtml"
+  url "https://github.com/jenniferlu717/Bracken/archive/v2.2.tar.gz"
+  sha256 "307495060d3575c3a90c87cf82074d4a63d20cf4062658f161951c60a66fd319"
+
+  depends_on "gcc" if OS.mac?
+
+  fails_with :clang # uses OpenMP
+
+  def install
+    cd "src" do
+      inreplace "Makefile", "g++", "$(CXX)"
```

Comment:
```suggestion
      inreplace "Makefile", "g++", ENV.cxx
```
Or you can delete this line altogether. It's handled by a feature of Homebrew called Superenv.

Reply:
Ah, interesting. For some reason I thought that my `brew install --interactive` wasn't working right.

Reply:
Use `which -a g++` and you should see that the first hit is a Homebrew shim script.